packages/opentelemetry-instrumentation-watsonx/pyproject.toml

```diff
@@ -11,11 +11,12 @@ readme = "README.md"
 include = "opentelemetry/instrumentation/watsonx"
 
 [tool.poetry.dependencies]
-python = ">=3.9,<4"
+python = ">=3.10,<4"
```

Comment:
@nirga If to set `python = ">=3.9` for ibm_watson_machine_learning by force, there may be potential risks.

Reply:
@jinsongo people won't be able to use it in Python 3.9 anyway since you can install `ibm-watson-machine-learning` on Python 3.9, right?

Reply:
@nirga, On Python 3.9, the highest supported version is 1.0.335, and Python 3.10 is required since 1.0.338
https://pypi.org/project/ibm-watson-machine-learning/1.0.347/#history
```
# python3 --version
Python 3.9.18
# pip install ibm-watson-machine-learning==1.0.347
ERROR: Could not find a version that satisfies the requirement ibm-watson-machine-learning==1.0.347 (from versions: 1.0.8, 1.0.9, 1.0.10, 1.0.14, 1.0.15, 1.0.22, 1.0.28, 1.0.29, 1.0.34, 1.0.38, 1.0.43, 1.0.44, 1.0.45, 1.0.51, 1.0.53, 1.0.57, 1.0.66, 1.0.70, 1.0.77, 1.0.79, 1.0.86, 1.0.91, 1.0.96, 1.0.99, 1.0.100, 1.0.102, 1.0.105, 1.0.115, 1.0.116, 1.0.118, 1.0.120, 1.0.122, 1.0.135, 1.0.139, 1.0.141, 1.0.142, 1.0.157, 1.0.162, 1.0.165, 1.0.166, 1.0.173, 1.0.175, 1.0.176, 1.0.178, 1.0.179, 1.0.180, 1.0.181, 1.0.189, 1.0.200, 1.0.202, 1.0.204, 1.0.206, 1.0.208, 1.0.209, 1.0.210, 1.0.214, 1.0.218, 1.0.222, 1.0.227, 1.0.229, 1.0.232, 1.0.237, 1.0.238, 1.0.244, 1.0.246, 1.0.248, 1.0.250, 1.0.253, 1.0.255, 1.0.256, 1.0.257, 1.0.260, 1.0.262, 1.0.264, 1.0.268, 1.0.272, 1.0.273, 1.0.277, 1.0.280, 1.0.283, 1.0.286, 1.0.288, 1.0.292, 1.0.293, 1.0.294, 1.0.297, 1.0.302, 1.0.307, 1.0.308, 1.0.309, 1.0.310, 1.0.311, 1.0.312, 1.0.313, 1.0.316, 1.0.319, 1.0.320, 1.0.321, 1.0.322, 1.0.324, 1.0.326, 1.0.327, 1.0.330, 1.0.333, 1.0.335)
```

Reply:
Let's use 1.0.335 for the tests then

Reply:
@nirga we can use https://pypi.org/project/ibm-watson-machine-learning/1.0.333/ for some test now.

Do we have any plan to upgrade the openLLMetry python version to a higher version or we want to stick to 3.9? Thanks

Reply:
We want to support a range from 3.9 to 3.12. At the end of the day, this doesn't affect your instrumentation, only the tests. So I don't see any reason not to use an older version if it's just for the tests.

Reply:
@nirga Please review, the ibm-watson-machine-learning 1.33 is added for tests. Thanks!